XRTK-Core/Assets/Plugins/Lumin/manifest.xml

```diff
@@ -6,6 +6,8 @@
     <uses-privilege ml:name="GesturesSubscribe" />
     <uses-privilege ml:name="GesturesConfig" />
     <uses-privilege ml:name="HandMesh" />
+    <uses-privilege ml:name="WorldReconstruction" />
+    <uses-privilege ml:name="Occlusion" />
```

Comment:
Does manifest automatically get used? Doesn't it have to be in the project root Assets/Plugins?

Reply:
It's the manifest used to test the build on device. It won't be included in anyone's project as it's not in the package folder.